app/assets/javascripts/helpers/requests.jsx

```diff
@@ -66,6 +66,26 @@
       });
     }
 
+    submit(route, formData) {
+      const request = new XMLHttpRequest();
+
+      request.open('POST', route);
+      request.setRequestHeader('Content-Type', 'multipart/form-data');
+      return new Promise(function (resolve, reject) {
+        request.onload = function () {
```

Comment:
Can we also do form validation somewhere along the way? Should only accept jpg/jpeg/png